packages/venia-ui/lib/components/App/renderRoutes.js

```diff
@@ -1,21 +0,0 @@
-import React, { lazy, Suspense } from 'react';
-import { Switch, Route } from '@magento/venia-drivers';
-import { Page } from '@magento/peregrine';
-import ErrorView from '../ErrorView/index';
-
-const CreateAccountPage = lazy(() => import('../CreateAccountPage/index'));
-const Search = lazy(() => import('../../RootComponents/Search'));
-
-const renderRoutingError = props => <ErrorView {...props} />;
-
-const renderRoutes = () => (
```

Comment:
We weren't using this at all? I wonder when we moved away from it and why it didn't get removed at that time.

Reply:
Not sure dude, that's why I was asking about it in the chat the other day.